backend/data/src/main/resources/I18n_en.properties

```diff
@@ -113,3 +113,16 @@ notifications.email.security-settings-link=Check your security settings <a href=
 notifications.email.mfa.MFA_ENABLED=Multi-factor authentication has been enabled for your account.
 notifications.email.mfa.MFA_DISABLED=Multi-factor authentication has been disabled for your account.
 notifications.email.password-changed=Password has been changed for your account.
+
+notifications.email.plan-migration-subject=Your 🐁 plan will be updated on {0}
+notifications.email.plan-migration-body=Hello! 👋<br/><br/>\
```

Comment:
Oh, the encoding is somehow broken. I believe utf-8 is now supported for properties file. Can we fix it? In the meantime, I am asking marketa to provide proper e-mail.